### ISSUE TITLE
[Feature request] Specifying different systems and facilities per reaction type

### ISSUE THREAD

**Issue by Maldark** [zmiguel/Eve-Reactions-Calculator#163]
Hi there,

First of all thanks a lot for the tool. It's quite amazing I must say.

I don't know if you accept feature requests, but I'll give it a shot and you can consider whether you think its something worthwhile.

I do reactions in two systems; composite and hybrid in one, and biochemical in another system. The first system has athanors with T2 rigs, and the second system has a Tatara with T2 rigs. 

Basically I'd love for there to be three separate sections, one for each type of reactions.
![image](https://github.com/user-attachments/assets/9875e381-97e7-430f-a629-ec40e659a7ab)

I'm obviously no designer but something along these lines.

Let me know what you think, thanks!

**Comment by Maldark** [comment]
Okay I don't know what happened to the picture, I just copied it from paint and it spazzed out. But you most likely get the point. One section for general settings, and three separate sections for Composite reactions, Hybrid reactions, and one for Biochemical reactions.

**Comment by zmiguel** [comment]
Hi! Sorry for the long response time, IRL has been busy and I missed the GH notification about this.

This should be possible, it requires some work, but I think I can squeeze it in this week

**Comment by Maldark** [comment]
Hi again,

Any chance for an update on the feature request :) ?

**Comment by zmiguel** [comment]
Hey @Maldark took me some time, but it's finally here! 
Now live in v2.5.0, let me know if this is what you had planned. I'll leave this issue open for a few more days in case you have any feedback!

**Comment by Maldark** [comment]
@zmiguel This is exactly what I wanted, thank you very much!
